src/Symfony/Component/Form/Form.php

```diff
@@ -68,7 +68,10 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
+ * @template T
+ *
  * @implements \IteratorAggregate<string, FormInterface>
```

Comment:
I suppose this also?
```suggestion
 * @implements \IteratorAggregate<string, FormInterface<T>>
```

Reply:
No, it is left like that on purpose because sub forms might have a different data type, so leave it without type parameter ( default mixed )